app/scripts/lib/ppom/ppom-middleware.ts

```diff
@@ -82,6 +86,19 @@ export function createPPOMMiddleware<
         return;
       }
 
+      if (req.method === MESSAGE_TYPE.ETH_SEND_TRANSACTION) {
+        const { to: toAddress } = req?.params?.[0] ?? {};
+        const internalAccount = accountsController.listAccounts();
```

Comment:
Minor, should this be `internalAccounts`?

Reply:
PR is updated.